recipes/cylc-flow/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4"
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - ansimarkup >=1.0.0
```

Comment:
Are these all needed in `host` as this section only needs to contain packages required by the `setup.py` script. Often just having `python` and `pip` is enough.

Reply:
I think it was my mistake when I wrote the first recipe. I simply copied most requirements to `host`. Is `host` equivalent to `setup_requires` in setuptools?

Reply:
Yes it is 👍

Reply:
In that case it should be just python, pip, and - I think - pytest-runner

Reply:
I'll remove all entries except those you have explicitly mentioned, @kinow. Thanks.

Reply:
... actually now I am doing that, I recall that I already removed ``pytest-runner`` as part of addressing Oliver's comments regarding ``pytest`` (see https://github.com/conda-forge/staged-recipes/pull/9661#discussion_r329170553). If you think it still needs to be there despite that, let me know. I assumed it was not needed, rlating only to the testing (from the name), so sorry if that was mistaken.

Reply:
I'll keep ``pytest-runner`` out, unless I am told otherwise.